src/data/java/com/fabriccommunity/spookytime/SpookyData.java

```diff
@@ -0,0 +1,24 @@
+package com.fabriccommunity.spookytime;
+
+import net.minecraft.data.DataGenerator;
+import net.minecraft.data.server.RecipesProvider;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+
+public final class SpookyData
+{
+	public static void main(String... args) throws IOException
+	{
+		Path path = Paths.get(args[0]);
+		DataGenerator dataGenerator = new DataGenerator(path, Collections.emptySet());
+		
+		dataGenerator.install(new SpookyRecipeProvider(dataGenerator));
+		
+		dataGenerator.run();
```

Comment:
I don't get how this is better than just writing the JSONs directly? It's not like we'll have thousands of them that need to be created en masse.

Reply:
Will add this back when we have a lot of jsons